### PR TITLE
neomutt 20211029

### PR DESCRIPTION
**Pull request by thezeroalpha**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/neomutt.rb [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
+
   license "GPL-2.0-or-later"
   head "https://github.com/neomutt/neomutt.git", branch: "master"
 
-  bottle do
-    sha256 arm64_big_sur: "d120a26ccf7566d068226b52c07040ad63364ccf5ffd9b69dbbc550d4966393b"
-    sha256 big_sur:       "98e1898da2c59d05bd043fd6b643b9a9b0ba9566acd7fa43e302b551463de7ed"
-    sha256 catalina:      "5f850eac5db2ea581b7b69db0bd1e28198fb483cb7c1a991e2448696b48354de"
-    sha256 x86_64_linux:  "b64849b42fd3b87dcec7c0db4393c2e8b024938fd0d947acc10bcbf78558ea58"
-  end
```
Comment (carlocab) [inline Formula/neomutt.rb:14]: Please don't touch the bottle block.

File: Formula/neomutt.rb [hunk 2]
```diff
@@ -1,18 +1,12 @@
 class Neomutt < Formula
   desc "E-mail reader with support for Notmuch, NNTP and much more"
   homepage "https://neomutt.org/"
-  url "https://github.com/neomutt/neomutt/archive/20211022.tar.gz"
-  sha256 "49aa5029665c6819e708276b9efa1ca71ec5afe870eb9f08e656107d234941e6"
+  url "https://github.com/neomutt/neomutt/archive/refs/tags/20211029.tar.gz"
+  sha256 "08245cfa7aec80b895771fd1adcbb7b86e9c0434dfa64574e3c8c4d692aaa078"
+
   license "GPL-2.0-or-later"
```
Comment (carlocab) [inline Formula/neomutt.rb:7]:
```suggestion
  url "https://github.com/neomutt/neomutt/archive/20211029.tar.gz"
  sha256 "08245cfa7aec80b895771fd1adcbb7b86e9c0434dfa64574e3c8c4d692aaa078"
  license "GPL-2.0-or-later"
```

File: Formula/neomutt.rb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 class Neomutt < Formula
   desc "E-mail reader with support for Notmuch, NNTP and much more"
   homepage "https://neomutt.org/"
-  url "https://github.com/neomutt/neomutt/archive/20211022.tar.gz"
-  sha256 "49aa5029665c6819e708276b9efa1ca71ec5afe870eb9f08e656107d234941e6"
+  url "https://github.com/neomutt/neomutt/archive/refs/tags/20211029.tar.gz"
```
Comment (carlocab) [inline Formula/neomutt.rb:4]:
```suggestion
  url "https://github.com/neomutt/neomutt/archive/20211029.tar.gz"
```
Do we really need the longer URL? They resolve to the same place. (See my previous suggestion.)

Comment (thezeroalpha) [reply Formula/neomutt.rb:4]: True, it seems that the shorter URL is enough.

**Review by carlocab — APPROVED** [review]
Thanks, @thezeroalpha!

**Pull request closed without merging**